pkg/common/terraform/terraform.go

```diff
@@ -0,0 +1,114 @@
+package terraform
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/hc-install/product"
+	"github.com/hashicorp/hc-install/releases"
+	"github.com/hashicorp/terraform-exec/tfexec"
+)
+
+type runner struct {
+	Context    context.Context
+	Installer  *releases.LatestVersion
+	Runner     *tfexec.Terraform
+	WorkingDir string
```

Comment:
```suggestion
	installer  *releases.LatestVersion
	runner     *tfexec.Terraform
	workingDir string
```

make all of these private and drop context

Reply:
Resolved in https://github.com/openshift/osde2e/pull/1558/commits/0de34542c4d705ab0513096a30d520fbf86d25bb